Source/ASTextNode.h

```diff
@@ -217,6 +217,15 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/**
+ * @abstract Text node unsupported properties
+ */
+@interface ASTextNode (Unsupported)
+
+@property (nonatomic) id _Nullable textContainerLinePositionModifier;
```

Comment:
```suggestion
@property (nullable, nonatomic) id textContainerLinePositionModifier;
```

Reply:
This `@protocol` was defined elsewhere (by YYTextLayout / ASTextLayout), so I don't think we can change it just here